packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -755,4 +755,28 @@ const api = new appsync.GraphqlApi(this, 'api', {
   definition: appsync.Definition.fromFile(path.join(__dirname, 'appsync.schema.graphql')),
   introspectionConfig: appsync.IntrospectionConfig.DISABLED,
 });
+```
+
+## Query Depth Limits
+
+There are times during which you may want more granular control over how the API functions during an operation. One such control is adding a limit to the amount of nested levels a query may process. By default, queries are able to process an unlimited amount of nested levels. Limiting queries to a specified amount of nested levels has potential implications for the performance and flexibility of your project.
```

Comment:
please split this up into a sentence per line so the raw markdown is more amenable to look at.

Reply:
Fixed